CMakeLists.txt

```diff
@@ -62,7 +70,7 @@ set_option(TRACY_FIBERS "Enable fibers support" OFF)
 set_option(TRACY_NO_CRASH_HANDLER "Disable crash handling" OFF)
 set_option(TRACY_TIMER_FALLBACK "Use lower resolution timers" OFF)
 
-if(BUILD_SHARED_LIBS)
+if(NOT TRACY_STATIC)
     target_compile_definitions(TracyClient PRIVATE TRACY_EXPORTS)
     target_compile_definitions(TracyClient PUBLIC TRACY_IMPORTS)
```

Comment:
Not related to the PR: I would expect this to be `INTERFACE` instead of `PUBLIC`.

https://cmake.org/cmake/help/latest/command/target_compile_definitions.html